netty/src/test/java/io/grpc/netty/NettyServerBuilderTest.java

```diff
@@ -36,9 +38,15 @@
 
   @Rule public final ExpectedException thrown = ExpectedException.none();
 
+  private NettyServerBuilder builder = null;
+
+  @Before
+  public void setUp() {
+    builder = NettyServerBuilder.forPort(8080);
```

Comment:
When possible, initialize variables at their declaration. There's no need for a `@Before` here. I'll fix this.